clients/privacy-center/cypress/support/commands.ts

```diff
@@ -79,7 +79,18 @@ declare global {
             Cypress.Withinable &
             Cypress.Shadow
         >
-      ): Chainable<JQuery<HTMLElement>>;
+      ): Chainable<boolean>;
+      /**
+       * Custom command to load a Privacy Center configuration JSON file from a fixture.
+       * 
```

Comment:
might be helpful to add a comment about the footgun here. fixtures normally persist across `.visit()`s, I guess in this case it isn't persisting because it's actually going into redux?

Reply:
I think it may actually be a side-effect of this, honestly: https://github.com/ethyca/fides/issues/3212...